spaces/S000184/properties/P000036.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000184
+property: P000036
+value: false
+---
+
+By construction as the union of two proper nonempty open sets.
```

Comment:
Need "disjoint union" instead of just "union".  Given "nonempty", can drop "proper" if we use "disjoint union".